src/frontend/context/Valutakurs/ValutakursSkjemaContext.tsx

```diff
@@ -0,0 +1,160 @@
+            );
+        }
+    });
+
+    const { skjema, valideringErOk, kanSendeSkjema, nullstillSkjema, onSubmit } = useSkjema<
+        IValutakurs,
+        IBehandling
+    >({
+        felter: {
+            periodeId: useFelt<string>({
+                verdi: valutakursFeilmeldingId(valutakurs),
+            }),
+            id: useFelt<number>({ verdi: valutakurs.id }),
+            status: useFelt<EøsPeriodeStatus>({ verdi: valutakurs.status }),
+            initielFom: initelFom,
+            barnIdenter: useFelt<OptionType[]>({
+                verdi: valgteBarn,
+                valideringsfunksjon: erBarnGyldig,
+            }),
+            periode: useFelt<IYearMonthPeriode>({
+                verdi: nyYearMonthPeriode(valutakurs.fom, valutakurs.tom),
+                avhengigheter: { initelFom },
+                valideringsfunksjon: erEøsPeriodeGyldig,
+            }),
+            valutakode: useFelt<string | undefined>({
+                verdi: valutakurs.valutakode,
+                valideringsfunksjon: erValutakodeGyldig,
+            }),
+            valutakursdato: useFelt<string | undefined>({
+                verdi: valutakurs.valutakursdato,
+                valideringsfunksjon: erValutakursDatoGyldig,
+            }),
+            kurs: useFelt<string | undefined>({
+                verdi: valutakurs.kurs,
+                valideringsfunksjon: erValutakursGyldig,
+            }),
+        },
+        skjemanavn: valutakursFeilmeldingId(valutakurs),
+    });
+
+    const sendInnSkjema = () => {
+        if (kanSendeSkjema()) {
+            onSubmit(
+                {
+                    method: 'PUT',
+                    data: {
+                        id: valutakurs.id,
+                        fom: skjema.felter.periode.verdi.fom,
+                        tom: skjema.felter.periode.verdi.tom,
+                        barnIdenter: skjema.felter.barnIdenter.verdi.map(barn => barn.value),
+                        valutakode: skjema.felter.valutakode?.verdi,
+                        valutakursdato: skjema.felter.valutakursdato?.verdi,
+                        kurs: skjema.felter.kurs?.verdi,
+                    },
+                    url: `/familie-ba-sak/api/differanseberegning/valutakurs/${behandlingId}`,
+                },
+                (response: Ressurs<IBehandling>) => {
+                    if (response.status === RessursStatus.SUKSESS) {
+                        nullstillSkjema();
+                        settEkspandertValutakurs(false);
+                        settÅpenBehandling(response);
+                    }
+                }
+            );
+        }
+    };
+
+    const slettValutakurs = () => {
+        request<void, IBehandling>({
+            method: 'DELETE',
+            url: `/familie-ba-sak/api/differanseberegning/valutakurs/${behandlingId}/${valutakurs.id}`,
+        }).then((response: Ressurs<IBehandling>) => {
+            if (response.status === RessursStatus.SUKSESS) {
```

Comment:
Samme som over ☝️

Reply:
lagt inn kode for at ved feil, så oppdateres submitRessurs og visFeilmeldinger